src/lib/realmdb/RealmDB.js

```diff
@@ -351,28 +351,22 @@ class RealmDB implements DB, ProfileDB {
 
   /**
    * get user profile from realmdb. result fields might be encrypted
-   * @param key
-   * @param field
+   * @param query
    * @returns {Promise<any | null>}
    */
   // eslint-disable-next-line require-await
-  async getProfileBy(query: object): Promise<Profile> {
+  async getProfileBy(query: Object): Promise<Profile> {
     return this.profiles.findOne(query)
   }
 
-  // eslint-disable-next-line require-await
-  async getProfilesBy(query: object): Promise<Array<Profile>> {
-    return this.profiles.find(query)
-  }
-
   /**
-   * Set profile fields
-   * @param fields
-   * @returns {Promise<Realm.Services.MongoDB.UpdateResult<any>>}
+   * get users profiles from realmdb. result fields might be encrypted
+   * @param query
+   * @returns {Promise<any | null>}
```

Comment:
doesnt match the return type

Reply:
fix with [d65c060](https://github.com/GoodDollar/GoodDAPP/pull/3391/commits/d65c060a3868c252b0641e1aa2f9c28d663e1efa)